keps/sig-scheduling/624-scheduling-framework/README.md

```diff
@@ -150,10 +143,10 @@ Scheduling cycles are run serially, while binding cycles may run concurrently.
 
 A scheduling cycle or binding cycle can be aborted if the pod is determined to
 be unschedulable or if there is an internal error. The pod will be returned to
-the queue and retried. If a binding cycle is aborted, it will trigger
-[Un-reserve](#un-reserve) plugins.
+the queue and retried. If a binding cycle is aborted, it will trigger the
+[Unreserve](#reserve) method in the Reserve plugin.
```

Comment:
In addition to this, can we move the tweaks in #1560 into this PR?

Reply:
done.